.github/workflows/publish.yaml

```diff
@@ -57,12 +54,23 @@ jobs:
               yaml.dump(citation, file, default_flow_style=False, sort_keys=False)
           "
 
-      - name: Commit and push changes done to the static version file and pyproject.toml
-        uses: actions-js/push@5a7cbd780d82c0c937b5977586e641b2fd94acc5 # pin@v1.5
+      - name: Update version in codemeta.json and URL
+        id: update-codemeta
+        env:
+          VERSION: ${{ steps.version.outputs.VERSION }}
+        run: |
+          # Ensure jq is installed
+          sudo apt-get install -y jq
+          # Update the version and URL in codemeta.json
+          jq --arg VERSION "$VERSION" --arg URL "https://github.com/ersilia-os/ersilia/v$VERSION" \
+             '.version = $VERSION | .codeRepository = $URL' codemeta.json > codemeta_tmp.json && mv codemeta_tmp.json codemeta.json
+
+      - name: Commit and push changes to CITATION.cff and codemeta.json
+        uses: actions-js/push@v1.5
         with:
           author_name: "ersilia-bot"
           author_email: "ersilia-bot@users.noreply.github.com"
-          message: "update version [skip ci]"
+          message: "Update version in CITATION.cff and codemeta.json [skip ci]"
```

Comment:
I guess we can just keep it as `Update version` because this step is updating a bunch of different files: `_static_version.py` in ersilia, `pyproject.toml`, the `CITATION.cff` file, and thanks to your work, the `codemeta.json` as well. We can add a descriptive comment in the beginning of the workflow to explain which files get updated in this pipeline.

Reply:
The commit message now uses "Update version"

Reply:
I added this comment at the start of the workflow to explain the the files being updated in the pipeline.
 _This job updates _static_version.py, pyproject.toml, CITATION.cff, and codemeta.json_